supplements/tomb-of-annihilation/items.xml

```diff
@@ -0,0 +1,183 @@
+           <p class="indent">The armor improves your combat readiness, granting you a +5 bonus to initiative as long as you aren't incapacitated.</p>
+           <p class="indent">The armor doesn't impose disadvantage on your Dexterity (Stealth) checks.</p>
+           <p class="indent">The armor doesn't impose disadvantage on saving throws made to resist the effects of extreme heat (see chapter 5 of the Dungeon Master's Guide).</p>
+           <p class="indent"><b><i>Curse.</i></b> This armor is cursed. Whenever you don or doff it, you must make a DC 15 Constitution saving throw, taking 100 (lOdlO + 45) poison damage on a failed save, or half as much damage on a s u ccessful one. Only a wish spell can remove the armor's curse.</p>
+       </description>
+       <setters>
+           <set name="category">Magic Armor</set>
+           <set name="cost" currency="gp">0</set>
+           <set name="type" addition="plate">Armor</set>
+           <set name="rarity">Rare</set>
+           <set name="armor">Plate</set>
+           <set name="attunement">true</set>
+       </setters>
+       <rules>
+           <stat name="initiative" value="5" />
+       </rules>
+   </element>
+
+<!-- STAFF OF THE FORGOTTEN ONE -->
+    <element name="Staff of the Forgotten One" type="Magic Item" source="Tomb of Annihilation" id="ID_WOTC_TOA_MAGIC_ITEM_STAFF_OF_THE_FORGOTTEN_ONE">
+        <description>
+            <p>This crooked staff is carved from bone and topped with the skull of a forgotten archmage whom Acererak destroyed long ago. Etched into the skull's forehead is Accrerak.'s rune, which is known on many worlds as a sign of death.</p>
+            <p class="indent"><b><i>Beneficial Properties.</i></b> While the staff is on your person, you gain the following benefits:</p>
+            <p class="indent">Your proficiency bonus to Intelligence (Arcana) and Intelligence (History) checks is doubled.</p>
+            <p class="indent">You can't be blinded, charmed, deafened, frightened, petrified, or stunned.</p>
+            <p class="indent">Undead with a challenge rating of 2 or lower will neither threaten nor attack you unless you harm them.</p>
+            <p class="indent">You can wield the staff as a +3 quarterstaff that deals an extra 10 (3d6) necrotic damage on a hit.</p>
+            <p class="indent"><b><i>Invoke Curse.</i></b> The Staff of the Forgotten One has 7 charges and regains ld4 + 3 expended charges daily at dawn. While holding the staff, you can use an action to expend 1 charge and target one creature you can see wit hin 60 feet of you. The target must succeed on a Constitution saving throw (using your spell save DC) or be cursed. While cursed in this way, the target can' t regain hit points and has vulnerability to necrotic damage. A greater restoration, remove c urse, or similar spell ends the curse on the target.</p>
```

Comment:
```suggestion
            <p class="indent"><b><i>Invoke Curse.</i></b> The Staff of the Forgotten One has 7 charges and regains 1d4 + 3 expended charges daily at dawn. While holding the staff, you can use an action to expend 1 charge and target one creature you can see within 60 feet of you. The target must succeed on a Constitution saving throw (using your spell save DC) or be cursed. While cursed in this way, the target can't regain hit points and has vulnerability to necrotic damage. A greater restoration, remove curse, or similar spell ends the curse on the target.</p>
```

Reply:
What exactly about 1d4 is changed. It looks the same to me?

Reply:
lower case L to a 1